src/assets/sass/protocol/base/elements/_quotes.scss

```diff
@@ -4,16 +4,24 @@
 
 @import '../../includes/lib';
 
-
 blockquote {
     @include bidi((
         (border-left, 5px solid $color-gray-20, border-right, 0),
         ));
-    @include text-display-md;
-    margin: $spacing-md auto;
+    @include text-display-sm;
+    margin: $spacing-lg auto $spacing-lg;
```

Comment:
When using the short hand like this we don't need to specify top and bottom separately if they're the same values.

When there's two values like this it reads as margin: top-and-bottom  left-and-right;

I think you know this though, since you didn't repeat the left-and-right values. So maybe this is left over from some other tweaking.

Reply:
Whoops! You're right.  👍